.gitignore

```diff
@@ -19,3 +19,4 @@ test/version_tmp
 tmp
 *.swp
 .ruby-version
+tags
```

Comment:
@joaomdmoura you don't use exuberant ctags?  such win
@leandrocp You can put tags in your global gitignore if this change doesn't make it in :)